app/models/filesystem.rb

```diff
@@ -120,6 +120,15 @@ def contents_displayable?
     !mime_type.binary?
   end
 
+  def displayable_contents
+    if contents_displayable? &&
+       ((contents[0].ord == 255 && contents[1].ord == 254) || (contents[0].ord == 254 && contents[1].ord == 255))
```

Comment:
@Fryguy You're right, what you're suggesting would be less complicated (and in truth was my first take
at the problem), though with this approach, the comparison using `==` wouldn't work, because -- as things are --
encoding of the `contents` and `"\xFF\xFE"` strings would be different.

The later would be utf-8, the former would be ascii.

I'm sure we could find a better solution than mine, but I just don't see it :-/